packages/plugins/debug-renderer/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@blockly/debug-renderer",
+  "version": "1.20200406.0",
+  "description": "A Blockly plugin that (TODO: add description)",
+  "scripts": {
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "dist": "blockly-scripts build prod",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run prod",
+    "start": "blockly-scripts start"
+  },
+  "main": "./dist/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "Blockly Team",
+  "keywords": [
+    "blockly",
+    "plugin",
+    "..."
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/master/packages/plugins/template#readme",
```

Comment:
Is it obvious that you have to change this? We decided to go with the name of template for the folder, but maybe that makes it too hidden in the wall of config.